fairing/deployers/serving/serving.py

```diff
@@ -41,7 +41,10 @@ def deploy(self, pod_spec):
         self.deployment = apps_v1.create_namespaced_deployment(self.namespace, self.deployment_spec)
         self.service = v1_api.create_namespaced_service(self.namespace, self.service_spec)
 
-        logger.warn("Endpoint {} launched.".format(self.deployment.metadata.name))
+        logger.warn("Deployment {} launched.".format(self.deployment.metadata.name))
+        url = "http://{0}.{1}.svc.cluster.local".format(self.service.metadata.name,
```

Comment:
This won't always be the service endpoint. the suffix `cluster.local` is configurable in Kubernetes

Reply:
Good to know. Is there a simple way to get the in cluster DNS name correctly? 

We could also return the empty string if you think that's better.